lib/miq_environment.rb

```diff
@@ -37,7 +37,7 @@ def self.supports_apache?
 
     def self.supports_systemd?
       return @supports_systemd unless @supports_systemd.nil?
-      @supports_systemd = is_appliance? && !is_container? && supports_command?('systemctl')
```

Comment:
Ugh this is what I get for trying to code and cook at the same time

Reply:
Burgers for dinner if anyone is curious haha so at least it is worth it

Reply:
😋